app/graphql/mutations/dossier_repasser_en_construction.rb

```diff
@@ -12,7 +12,7 @@ class DossierRepasserEnConstruction < Mutations::BaseMutation
     field :errors, [Types::ValidationErrorType], null: true
 
     def resolve(dossier:, instructeur:, disable_notification:)
-      dossier.repasser_en_construction!(instructeur)
+      dossier.repasser_en_construction!(instructeur: instructeur, disable_notification: disable_notification)
```

Comment:
dans la lignée du `max:` ça pourrait s'écrire aussi :

```suggestion
      dossier.repasser_en_construction!(instructeur:, disable_notification:)
```

Reply:
j'ai ajouté un commit qui reprend tout les mutations avec la syntaxe plus concise